index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Capstone project 1</title>
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+  <script src="https://kit.fontawesome.com/e8963cdebe.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <nav>
+      <img class="logo" src="assets/icons/logo.svg" alt="Logo"> <!-- Placeholder logo from https://logoipsum.com-->
+      <i class="fa fa-bars fa-lg js-humburger-menu-icon"></i>
+      <ul class="menu">
+        <li><a class="button--secondary js-home-page-link" href="#">Home</a></li>
+        <li><a class="button--secondary js-about-page-link" href="#">About</a></li>
+      </ul>
+      <a class="button--primary" href="#">Book a date</a>
+    </nav>
```

Comment:
- [ ] Kindly add the top short dark bar section that contains the social media links, logout, and language links.